app/assets/javascripts/oxalis/model/bucket_data_handling/pullqueue.js

```diff
@@ -148,7 +148,6 @@ class PullQueue {
   ): void {
     const zoomStep = bucketAddress[3];
     const bucket = this.cube.getBucket(bucketAddress);
-    this.cube.boundingBox.removeOutsideArea(bucket, bucketAddress, bucketData);
```

Comment:
This is the most notable change. Currently, we are checking for out-of-boundingbox in the shader, so there's no need to (costly) rewrite the bucket data.